minecraft_servers/heavy/manifest.json

```diff
@@ -0,0 +1,45 @@
+{
+  "server_name": "heavy",
+  "nice_name": "Rede Heavy",
+  "direct_ip": "redeheavy.com.br",
+  "server_wildcards": [
+    "%.redeheavy.com.br"
```

Comment:
Not a real wildcard
https://i.imgur.com/Z4xHsDU.png

Reply:
Okay, it's been resolved with DNS